README.md

```diff
@@ -1,3 +1,8 @@
+# Fork changes
+
+Added lowercased candidates comparison, Damerau distance, Bayesian reasoning, GPU inference. Exposed top_n candidates parameter instead of the hardcoded number.
+As explained in the original <a href="https://github.com/R1j1t/contextualSpellCheck/issues/88/">issue</a>.
+
```

Comment:
You will need to remove this

Reply:
Indeed, I did not realize that at first ) Should I add a brief explanation of the added functionality in the Usage section instead, or just leave readme.md untouched?